cmd/pw_ws_broker/web.go

```diff
@@ -293,6 +293,10 @@ func (c *WsClient) planeProtocolHandler(ctx context.Context, conn *websocket.Con
 		grid[k+"_high"] = true
 	}
 
+	locationMessages := make([]*export.PlaneLocation, 0, 1000)
+	sendTick := time.NewTicker(100 * time.Millisecond)
```

Comment:
Can we make this configurable? Could be a useful point for tuning.

Reply:
I have included a command line option for configuring this.

by default it is off and we will send each message as we get it.

we can set it to 100ms to see if that helps